python/tests/catalog/test_glue.py

```diff
@@ -0,0 +1,58 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+from unittest.mock import patch, MagicMock
+
+import botocore.exceptions
+
+from pyiceberg.catalog import PropertiesUpdateSummary
+from pyiceberg.catalog.glue import GlueCatalog
+from pyiceberg.schema import Schema
+
+import random
+import string
+
+
+def get_randam_table_name():
```

Comment:
Thank you for your suggestions. The current `test_glue.py` is a placeholder to remind me to add unit tests later. Since `GlueCatalog` is still under development, I will keep the temporal tests for a moment for convenient verification of my changes to the code. I will change them to actual unit tests based on what you suggested once we finish reviewing the `create_table` and `load_table`.

Reply:
sound good, we should definitely keep the files around, because ideally we want to also do integration tests like what we did in Java.